app/src/main/java/org/simple/clinic/user/UserSession.kt

```diff
@@ -215,6 +228,15 @@ class UserSession @Inject constructor(
         .doOnSuccess { Timber.i("Registration result: $it") }
   }
 
+  private fun reportUserRegisteredToAnalytics() {
+    loggedInUser()
+        .subscribeOn(schedulersProvider.io())
+        .take(1)
+        .filterAndUnwrapJust()
+        .map(Analytics::setNewlyRegisteredUser)
+        .subscribe()
```

Comment:
Same here. Dangling `subscribe` call.

Reply:
Same answer as the previous comment. I'll do the same here.